### PR TITLE
Improve Slovak translation

### PR DESCRIPTION
**Pull request by Msprg**
### Purpose

I know that translations should be done via website, but this was much faster and quicker approach to fix the translation errors. Sorry about that, hope it's not much trouble.

## Authorship

Your name and email will be added automatically to the AUTHORS file
based on the commit metadata.

### CODE REVIEW

**Comment by acolomb** [comment]
I don't mind merging the changes here, but there won't be much review I suppose.  The number of people watching this repo is small, but even fewer among them speak Slovak.  This and many other aspects will be better handled on Weblate.

Please if you can, sign up there (https://hosted.weblate.org/projects/syncthing/ - can even use / connect your GitHub login) and contribute.  I find the interface quite pleasant to work with, and there will be a place for comments / discussion per string, automatic suggestions, etc.  Many features that you and other Slovak users might benefit from if you want to keep contributing.  There will be a large-ish GUI change in the near future when the WebAuthn stuff is merged, with quite some new strings to translate.

In short: Looking forward to seeing you on Weblate, also for future on the Slovak translation!

**Comment by calmh** [comment]
Yeah, no, translations need to come through the translation UI, sorry.
